src/google_map_markers.js

```diff
@@ -16,14 +16,25 @@ const mainStyle = {
 };
 
 const style = {
-  width: 0,
-  height: 0,
   left: 0,
   top: 0,
   backgroundColor: 'transparent',
   position: 'absolute',
 };
 
+const markerAxisX = {
+  left: 0,
+  center: '-50%',
+  right: '-100%',
+};
+
+const markerAxisY = {
+  top: 0,
+  center: '-50%',
+  bottom: '-100%',
+};
+const isNotProduction = process.env.NODE_ENV !== 'production';
```

Comment:
Why do we need this? :)

Reply:
We are using it to determine whether to console.error

Reply:
Trying to match how proptypes works

Reply:
Ok, sounds good! We could call it `isProduction` instead, and then do `if (!isProduction)`

Reply:
Alright :) I will make that update